src/store/store.js

```diff
@@ -0,0 +1,17 @@
+import {useState} from 'react';
+
+const useGlobalState = () => {
+  const [loggedIn, setIsLoggedIn] = useState(false);
+
+  const handleIsLoggedIn = (bool) => {
+    setIsLoggedIn(bool);
+  };
+
+  function isLoggedIn() {
+    return loggedIn;
+  }
+
+  return {isLoggedIn, handleIsLoggedIn};
```

Comment:
Podes retornar a variável **loggedIn** no objeto do state, não precisas de criar uma função que retorna a variável.
E acho que pode retornar a função do setState também
```suggestion
  return {loggedIn, setIsLoggedIn};
```

Reply:
Makes sense!